### PR TITLE
* Migrated from `@supabase/supabase-auth-helpers` to `@supabase/auth-helpers`

### PR DESCRIPTION
**Pull request by agrittiwari**
## What kind of change does this PR introduce?

What I did:

* Updated `readme.md` for usage with screenshots and direction to use supabase Oauth.
----
* Installed `@supabase/ui` `@supabase/supabase-js`, `@supabase/auth-helpers-nextjs` and `@supabase/auth-helpers-react` libraries from npm.
*  Replaced all imports of `@supabase/supabase-auth-helpers/nextjs` in your project with `@supabase/auth-helpers-nextjs`.
* Replaced all imports of `@supabase/supabase-auth-helpers/react` in your project with `@supabase/auth-helpers-react`.
* Replaced all instances of `withAuthRequired` in any of your NextJS pages with `withPageAuth`.
* Replaced all instances of `withAuthRequired` in any of your NextJS API endpoints with `withApiAuth`.
* Uninstalled `@supabase/supabase-auth-helpers`.

## What is the current behavior?

It was using the old package names for supabase-auth helpers

## What is the new behavior?

This includes new packages as it is released. [https://github.com/supabase-community/auth-helpers/tree/main/examples/nextjs](https://github.com/supabase-community/auth-helpers/tree/main/examples/nextjs)

## Additional context

Add any other context or screenshots.
@joshenlim Please review this

### CODE REVIEW

File: examples/nextjs-with-supabase-auth/pages/_app.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-import './../style.css'
+// pages/_app.js
+import React from 'react';
+import { UserProvider } from '@supabase/auth-helpers-react';
+import { supabaseClient } from '@supabase/auth-helpers-nextjs';
 
-export default function MyApp({ Component, pageProps }) {
+export default function App({ Component, pageProps }) {
   return (
-    <main className={'dark'}>
```
Comment (silentworks) [inline examples/nextjs-with-supabase-auth/pages/_app.js:7]: Any particular reason why this div with the dark class was removed?

File: examples/nextjs-with-supabase-auth/pages/profile.js [hunk 2]
```diff
@@ -1,13 +1,8 @@
 // pages/profile.js
-import { withAuthRequired } from '@supabase/supabase-auth-helpers/nextjs'
+import { withPageAuth } from '@supabase/auth-helpers-nextjs';
 
 export default function Profile({ user }) {
-  return (
-    <>
-      <div>Hello {user.email}</div>
-      <pre>{JSON.stringify(user, null, 2)}</pre>
```
Comment (silentworks) [inline examples/nextjs-with-supabase-auth/pages/profile.js:8]: Any particular reason why the pre tag and its content was removed?

**Review by silentworks — CHANGES_REQUESTED** [review]
Please configure your editor to remove semicolons, as its marking come existing code as changes but aren't necessary.

File: examples/nextjs-with-supabase-auth/pages/_app.js (hunk 1 above)
Comment (agrittiwari) [reply examples/nextjs-with-supabase-auth/pages/_app.js:7]: Yes, the dark background was affecting the readability after signup.

File: examples/nextjs-with-supabase-auth/pages/profile.js (hunk 2 above)
Comment (agrittiwari) [reply examples/nextjs-with-supabase-auth/pages/profile.js:8]: It was done out of testing, now back in the code.

**Review by silentworks — APPROVED** [review]
LGTM.

**Comment by agrittiwari** [comment]
> LGTM.

How do I ask for merge?

**Comment by silentworks** [comment]
@agrittiwari the errors are probably related to the conflicting files below. You will need to resolve those before the automatic merge can take place.

**Comment by agrittiwari** [comment]
> @agrittiwari the errors are probably related to the conflicting files below. You will need to resolve those before the automatic merge can take place.

Hey, leerob says that Supabase team needs to authorize.
Link to the discussion is here. [https://github.com/vercel/vercel/discussions/8337](https://github.com/vercel/vercel/discussions/8337)

**Comment by MildTomato** [comment]
Looks like conflicts need resolving.

**Comment by agrittiwari** [comment]
> Looks like conflicts need resolving.

@MildTomato Now, there is only one failing check and there are no conflicts to resolve as I removed the lock files.

**Pull request closed without merging**